modules/cnf-nrop-support-schedulable-resources.adoc

```diff
@@ -16,8 +16,20 @@ The NUMA Resources Operator provides topology-aware scheduling for workloads tha
 
 When configuring the NUMA Resources Operator, its management scope is determined by the `nodeGroups` field in its custom resource (CR). This principle applies to both compact and multi-node clusters.
 
-Compact clusters:: In a compact cluster, all nodes are configured as schedulable control plane nodes. The NUMA Resources Operator can be configured to manage all nodes in the cluster. Follow the deployment instructions for more details on the process. 
+Compact clusters:: In a compact cluster, all nodes are configured as schedulable control plane nodes. The NUMA Resources Operator can be configured to manage all nodes in the cluster. Follow the deployment instructions for more details on the process.
 
 Multi-Node OpenShift (MNO) clusters:: In a Multi-Node {product-title} cluster, control plane nodes are made schedulable in addition to existing worker nodes. To manage these nodes, you can configure the NUMA Resources Operator by defining separate `nodeGroups` in the `NUMAResourcesOperator` CR for the control plane and worker nodes. This ensures that the NUMA Resources Operator correctly schedules pods on both sets of nodes based on resource availability and NUMA topology.
 
+[NOTE]
+====
+The presence of stricter pod disruption budgets (PDBs) on control plane nodes means that profile modifications resulting in node reboots carry a higher risk of the machine config pool (MCP) becoming stuck if protected pods are unhealthy. The cluster's resilience mechanisms, while necessary, can lead to a stuck MCP if essential pods cannot be safely evicted.
```

Comment:
For your consideration:
Broke up first sentence to reduce length.
Removed last sentence as it's repetition?

```suggestion
Modifying a performance profile can trigger control plane node reboots. Stricter pod disruption budgets (PDBs) on control plane nodes increase the risk of the machine config pool (MCP) becoming stuck during the reboot process if protected pods are unhealthy.
```

Reply:
THanks I still want to retain some idea of what PDB is so I left that in